tests/qchem/of_tests/test_molecular_hamiltonian.py

```diff
@@ -401,6 +700,44 @@ def test_real_hamiltonian(symbols, geometry, method, args, tmpdir):
     assert np.isrealobj(hamiltonian.terms()[0])
 
 
+@pytest.mark.parametrize(
+    ("symbols", "geometry", "method", "args"),
```

Comment:
Since `symbols` and `geometry` are same here, we can shift them to function itself.